presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -154,6 +162,7 @@ protected InternalResourceGroup(Optional<InternalResourceGroup> parent, String n
             id = new ResourceGroupId(name);
             root = this;
         }
+        this.isStatic = isStatic;
     }
 
     public ResourceGroupInfo getFullInfo()
```

Comment:
How about changing this method signature to something like `public ResourceGroupInfo getFullInfo(boolean includeQueryInfo, boolean summarizeSubgroups, boolean includeDynamicSubgroups)`?